generator/src/main/java/com/scottlogic/deg/generator/generation/GenerationConfig.java

```diff
@@ -172,6 +184,13 @@ public String toString() {
             public static final String DEFAULT = CSV;
         }
 
+        public static class Destinations {
```

Comment:
Could we use enums here?

Reply:
Suggest removing these constants based on other comments